public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="eng">
+<head>
+  <title>Airbnb Proxy</title>
+</head>
+<body>
+    <div id="location"></div>
+    <div id="about-modal"></div>
+    <div id="description"></div>
+    <div id="description-modal-root"></div>
+    <div id="reservation"></div>
+    <div id="review"></div>
+    <div id="reviewmodal-root"></div>
+    <script crossorigin src ="http://localhost:2001/bundle.js"></script>
+    <script crossorigin src ="http://localhost:4000/bundle.js"></script>
```

Comment:
this is something I should look into crossorgin src. is that supposed to help with the cors policty issues?

Reply:
no, i actually deleted it. i changed it in my next commit